### ISSUE TITLE
[CDP] Incorporate accessibility feedback from the midpoint review: 'Check details' link

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#42872]
## Background

We received the following feedback regarding the 'Check details' link during the Combined Debt Portal's midpoint review: 

> On the Current VA Debt page when there are current debts (eg. the No copays on record view), the "Check details" link text for current debts may be a little ambiguous. Two big potential impacts here are screen reader users (harder to differentiate which link goes where) and voice command users (speaking the command Click check details link will match with both links). Consider adding some extra text to make the link purpose a little clearer across every possible use case, eg. "Check details" becomes "Check compensation and pension debt details." If that's not a practical option, then when this goes to code we may still need some helping aria-label text for screen reader users.

We previously met with Angela Fowler and discovered that screen readers should be reading out the full string as the feedback specifies, but it wasn't working as intended. 

## Solution

Angela's suggestion is that we use the `aria describe by` label. 

--
## Tasks
- [ ] Use the `aria describe by` label for the 'Check details' link under the copay and debt pages 
   - The copay version should read out the full facility name. IE 'Check details and resolve this debt for [VA facility name]`
   - The debt version should read out the full debt type. IE 'Check details and resolve this compensation and pension debt'

## Acceptance Criteria
- [ ] The `aria describe by` label has been incorporated 
- [ ] Reach out to Angela Fowler to check staging to make sure it's reading out on a screen reader as intended
